src/main/java/com/github/tomakehurst/wiremock/matching/AbstractDateTimeMatchResult.java

```diff
@@ -83,8 +85,7 @@ public double getDistance() {
   }
 
   private double calculateDistance(Temporal start, Temporal end) {
-    double distance = ((double) ChronoUnit.YEARS.between(start, end)) / 100;
-    distance = Math.abs(distance);
-    return Math.min(distance, 1.0);
+    long absoluteTimeDifference = Math.abs((ChronoUnit.MILLIS.between(start, end)));
+    return (double) absoluteTimeDifference / (absoluteTimeDifference + 50 * ONE_YEAR_IN_MILLIS);
```

Comment:
Having played around with this in a spreadsheet a bit I suggest changing the 50 to 2. This means a 10 year difference gives a distance of 0.83 and 100 years 0.98 which seems about right as a tradeoff for more range with smaller differences.

Otherwise, great stuff - exactly what I had in mind!

Reply:
Suggestion implemented in amended commit.